ouroboros-consensus-test/src/Test/LedgerTables.hs

```diff
@@ -0,0 +1,63 @@
+{-# LANGUAGE FlexibleContexts #-}
+
+module Test.LedgerTables (
+    prop_stowable_laws
+  , prop_tablestuff_laws
+  ) where
+
+import           Data.Function (on)
+
+import           Ouroboros.Consensus.Ledger.Basics
+import           Ouroboros.Consensus.Ledger.Tables
+
+import           Test.QuickCheck
+
+-- | We will compare the Ledger Tables of the result because the comparison for
+-- the rest of the LedgerState takes considerably more time to run.
+compareTables ::
+     ( IsMapKind mk
+     , TableStuff (LedgerState blk)
+     , Eq (LedgerTables (LedgerState blk) mk)
+     , Show (LedgerTables (LedgerState blk) mk)
+     )
+  => LedgerState blk mk
+  -> LedgerState blk mk
+  -> Property
+compareTables = (===) `on` projectLedgerTables
+
+-- | The StowableLedgerTables instances should follow these two laws:
+--
+-- > stow . unstow == id
+--
+-- > unstow . stow == id
+prop_stowable_laws ::
+     ( TableStuff (LedgerState blk)
+     , StowableLedgerTables (LedgerState blk)
+     , Eq (LedgerTables (LedgerState blk) EmptyMK)
+     , Show (LedgerTables (LedgerState blk) EmptyMK)
+     , Show (LedgerTables (LedgerState blk) ValuesMK)
+     )
+  => LedgerState blk EmptyMK
+  -> Property
+prop_stowable_laws = \ls ->
+  compareTables (stowLedgerTables (unstowLedgerTables ls)) ls .&&.
```

Comment:
If we'd introduce an operator for `compareTables` maybe this could be rewritten as:

```haskell
stowLedgerTables (unstowLedgerTables ls) ==? ls
```

or alternatively:

```haskell
stowLedgerTables (unstowLedgerTables ls) `compareTables` ls
```